src/components/Navbar/NavbarBody/index.tsx

```diff
@@ -49,13 +48,16 @@ const NavbarBody: React.FC = () => {
             </Button>
             <NavigationDrawer />
           </>
-          <Link href="/">
-            <a>
-              <Button shape={ButtonShape.Circle} variant={ButtonVariant.Ghost}>
-                <IconQ />
-              </Button>
-            </a>
-          </Link>
+          <Button
+            href="/"
+            shape={ButtonShape.Circle}
+            variant={ButtonVariant.Ghost}
+            className={styles.logoWrapper}
+            size={ButtonSize.Large}
```

Comment:
- using button size large, following the design (Q icon is bigger than the menu icon). 
- But there is a side effect, try hovering on menu icon and Q icon. 
  - The sizing of the  background color will bit slightly different